changelogs/unreleased/gh-7897-sending-arbitrary-iproto-packets.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
```

Comment:
AFAIC, we use normal past tense narration in changelogs — in commit messages we use imperative mood present tense[^1]. 

As for the commit subject, AFAIS, its goal is to summarize what the patch is about[^2] — I couldn't come up with an imperative mood description (i.e., `box: {implement, add, allow} *feature*` sounds awkward to me).

I couldn't come with a suitable imperative for the commit subject, i.e., I think `implement sending of arbitrary IPROTO packets` sounds awkard. If you have any ideas 

[^1]: https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message, point 6
[^2]: https://docs.kernel.org/process/submitting-patches.html#the-canonical-patch-format

Reply:
My comment wasn't about the changelog (GitHub doesn't allow to leave comments for commit message, so I had to write it for some empty line in the changelog file). As for the message - no, we use imperative only for title. By your [1] link you can see `"Use the imperative mood in the subject line."`. The subject line (title), not the entire message. I don't have any suggestions though. You can keep it any way you want.

Reply:
Sorry, I indeed misread our guideline. But all this time I was also using imperative mood present tense in commit messages, following this idea:
> Describe your changes in imperative mood, e.g. “make xyzzy do frotz” instead of “[This patch] makes xyzzy do frotz” or “[I] changed xyzzy to do frotz”, as if you are giving orders to the codebase to change its behaviour.[^1]

To sum up, I'd rather leave the commit subject as it is.

[^1]: https://docs.kernel.org/process/submitting-patches.html#the-canonical-patch-format

Reply:
We are not kernel though. Anyway ...